pymodm/fields.py

```diff
@@ -1093,11 +1095,30 @@ def to_mongo(self, value):
         return self._model_to_document(value)
 
 
-class EmbeddedDocumentListField(RelatedEmbeddedModelFieldsBase):
-    """A field that stores a list of documents within a document.
+class EmbeddedDocumentField(_EmbeddedDocumentFieldBase):
+    """**DEPRECATED**: A field that stores a document inside another document."""
+    def __init__(self, model,
+                 verbose_name=None, mongo_name=None, **kwargs):
+        warnings.warn('EmbeddedDocumentField is deprecated. Use '
+        'EmbeddedModelField instead.', DeprecationWarning, stacklevel=2)
+        super(EmbeddedDocumentField, self).__init__(model=model,
+                                                    verbose_name=verbose_name,
+                                                    mongo_name=mongo_name,
+                                                    **kwargs)
 
-    All documents in the list must be of the same type.
-    """
+
+class EmbeddedModelField(_EmbeddedDocumentFieldBase):
+    """A field that stores a document inside another document."""
+    def __init__(self, model,
+                 verbose_name=None, mongo_name=None, **kwargs):
+        super(EmbeddedModelField, self).__init__(model=model,
+                                                    verbose_name=verbose_name,
+                                                    mongo_name=mongo_name,
+                                                    **kwargs)
```

Comment:
Do we even need the __init__ method here? It is the same thing as the base class so I think we can remove it entirely.